homeassistant/components/qvr_pro/__init__.py

```diff
@@ -62,6 +65,9 @@ def setup(hass, config):
     except AuthenticationError:
         _LOGGER.error("Authentication failed")
         return False
+    except ConnectionError:
+        _LOGGER.error("Error connecting to QVR server.")
```

Comment:
We don't end logging messages with period.
```suggestion
        _LOGGER.error("Error connecting to QVR server")
```